analyzers/tests/SonarAnalyzer.Test/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Binary.cs

```diff
@@ -780,6 +770,39 @@ empty switch
             };
     }
 
+    [DataTestMethod]
+    [DataRow("list.Count == -5", true)]    // if should be unreachable
+    [DataRow("list.Count != -5", false)]   // else should be unreachable
+    [DataRow("list.Count <  -5", true)]    // if should be unreachable
+    [DataRow("list.Count >  -5", false)]   // else should be unreachable
+    [DataRow("list.Count <   0", true)]    // if should be unreachable
+    [DataRow("list.Count >=  0", false)]   // else should be unreachable
+    [DataRow("-5 == list.Count", true)]    // if should be unreachable
+    [DataRow("-5 != list.Count", false)]   // else should be unreachable
+    [DataRow("0 >   list.Count", true)]    // if should be unreachable
+    [DataRow("0 <=  list.Count", false)]   // else should be unreachable
```

Comment:
Remove the comments. They were there to show a gap in the engine, which you fixed.